04-Continous-Assurance/Readme.md

```diff
@@ -49,6 +49,12 @@ AzSK adds the service principal runtime account as a 'Reader' to the subscriptio
 
 2. Target OMS WorkspaceID* and SharedKey. (The OMS workspace can be in a different subscription, see note below)
 
+**Prerequisite:**
+
+**1.** The machine should be running on one of the below OS: 	
```

Comment:
Better to mention 'We currently support following OS options-' rather than saying 'machine should be running' :)

Reply:
Resolved